### DISCUSSION TITLE
标题选中非正文的时候，上传图片被禁用了

### DISCUSSION THREAD

**Discussion by nuaajiangteng** [wangeditor-next/wangEditor-next#301] [Q&A] [answered]
标题选中非正文的时候，上传图片被禁用了，是bug还是设计如此
![10](https://github.com/user-attachments/assets/f6f43505-08a7-4cb3-9a9a-f474b748d8b8)

**Comment by cycleccc** [comment, marked as answer]
有意做的，在标题中插入图片不符合文档编写规范。
https://github.com/cycleccc/wangEditor-next/blob/af62f6920a2f78ecda8a1fbd39a678a85ca06c0f/packages/basic-modules/src/modules/image/helper.ts#L30
